workflow/labels.md

```diff
@@ -43,7 +44,9 @@ Each review label has 3 steps:
 | ------- | ------------------------------------------------------------------------------------------------------------------------------ |
 | Blocked | Alerts managers that unblocking it requires their help. Any blocked issue should have a comment explaining what help you need. |
 | Doing   | Actually working on the issue                                                                                                  |
-| Triaged | A team member has reviewed this issue, agreed that it is valid and assigned a priority                                         |
+| Triaged | A team member has reviewed this issue, agreed that it is valid and assigned a priority                           | Invalid | This isn't a bug. |
```

Comment:
- The missing newline here is causing GitHub’s Markdown parser to [omit the “Invalid” row altogether](https://github.com/canonical-web-and-design/practices/blob/pmahnke-patch-1/workflow/labels.md).
- Though it’s changed here accidentally, I’m not aware of any repo that actually has a “Triaged” label. Which is just as well, since the definition of triaged is that it has a priority, which is determined by whether it has a priority label. Labelling an issue as “Triaged” would do nothing to triage it, it would just be noise.
- Currently many/most “Invalid” labels are applied by stalebot, which uses it to mean “inactive”, independent of whether the issue is valid, a bug, both, or neither. If we keep stalebot, we should tweak it so that it uses a different label (like, say, “stale”).

Reply:
I agree, my understanding is that "triaged" means "has been assigned a priority", and so that should be indicated by the fact of it having been assigned a priority.

I also think we should strive to use [simple English](https://en.wikipedia.org/wiki/Simple_English_Wikipedia) where possible, so it would be nice if we found other ways to communicate things like "triaged" - and talking in terms of priorities instead is a step in that direction.

Reply:
removing triaged

Reply:
actually, Triaged isn't a label in the webteam, it is a column on zenhub, so let's remove it.